crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs

```diff
@@ -849,3 +951,67 @@ fn is_airflow_builtin_or_provider(segments: &[&str], module: &str, symbol_suffix
         _ => false,
     }
 }
+
+fn uses_removed_context_keys(checker: &mut Checker) -> bool {
```

Comment:
Is this function only check function with `@task`? we probably need some docstring and make the function name more readable.

Reply:
I have renamed and added the docstring